coalib/results/ResultFilter.py

```diff
@@ -98,7 +112,7 @@ def source_ranges_match(original_file_dict,
         # original file with file_diff and original_diff applied
         original_total_file = original_total_diff.modified
         # modified file with modified_diff applied
-        modified_total_file = modified_result_diff_dict[file_name].modified
+        modified_total_file = modified_result_diff_dict[mod_file_name].modified
```

Comment:
I'd use the same `renamed_files.get(file_name, file_name)` here, no need for the one variable for one use